src/app.css

```diff
@@ -147,4 +138,22 @@
   .ul-leather li a {
     @apply text-gray-800 hover:text-primary-400 dark:text-gray-300 dark:hover:text-primary-500;
   }
+  .network-link-leather {
+    @apply stroke-gray-400 fill-gray-400;
+  }
+  .network-node-leather {
+    @apply stroke-gray-800;
+  }
+  .network-node-content {
+    @apply fill-[#d6c1a8];
+  }
+}
+@layer components {
+  .leather-legend {
```

Comment:
I'd suggest we rename this to `.legend-leather` to fit with the scheme of the other style classes.